vulnfeeds/cmd/nvd-cve-osv/run_cve_to_osv_generation.sh

```diff
@@ -69,18 +69,18 @@ done
 # Copy (and remove any missing) results to GCS bucket, with some sanity
 # checking.
 objs_present=$(gsutil ls "${OSV_OUTPUT_GCS_PATH}" | wc -l)
-objs_deleted=$(gsutil -m rsync -c -n -d "${WORK_DIR}/nvd2osv/gcs_stage" "${OSV_OUTPUT_GCS_PATH}" 2>&1 | grep "Would remove" | wc -l)
+objs_deleted=$(gsutil -m rsync -c -n "${WORK_DIR}/nvd2osv/gcs_stage" "${OSV_OUTPUT_GCS_PATH}" 2>&1 | grep "Would remove" | wc -l)
 
 threshold=$(echo "scale=2; ${objs_present} * (${SAFETY_THRESHOLD_PCT:-2} / 100)" | bc)
 
 # Bash can't deal with floats
 if (( $(echo "${objs_deleted} > ${threshold}" | bc -l) )); then
   echo "Aborting. Unexpectedly high (${objs_deleted}) number of CVE records would be deleted!" >> /dev/stderr
-  gsutil -m rsync -c -n -d "${WORK_DIR}/nvd2osv/gcs_stage" "${OSV_OUTPUT_GCS_PATH}" 2>&1 | grep "Would remove" >> /dev/stderr
+  gsutil -m rsync -c -n "${WORK_DIR}/nvd2osv/gcs_stage" "${OSV_OUTPUT_GCS_PATH}" 2>&1 | grep "Would remove" >> /dev/stderr
   exit 1
 fi
```

Comment:
I think we should leave some logs in - the `-n` flag is for dry-run mode.

Maybe just remove the `exit 1` (and change the wording of the logs?)